etc/kayobe/kolla.yml

```diff
@@ -37,7 +37,7 @@ kolla_ansible_source_url: https://github.com/stackhpc/kolla-ansible.git
 
 # Version (branch, tag, etc.) of Kolla Ansible source code repository if type
 # is 'source'.
-kolla_ansible_source_version: refs/tags/stackhpc/6.1.0.6
+kolla_ansible_source_version: stackhpc/rocky
```

Comment:
Can't really merge with this in place until we upgrade the rest.

Reply:
Yeah, i'm just working off this branch for now and will use the relevant tags for the upgrade.